internal/kernel_default.h

```diff
@@ -80,12 +80,16 @@ GEMMLOWP_SET_DEFAULT_KERNEL(false, false, SSE4_32_Kernel4x4Depth2)
 GEMMLOWP_SET_DEFAULT_KERNEL(false, false, SSE4_64_Kernel12x4Depth2)
 #else
 #ifndef GEMMLOWP_ALLOW_SLOW_SCALAR_FALLBACK
+#if defined __ARM_ARCH_5TE__
+#message "SIMD is not available on this platform. Using slow fallback."
```

Comment:
I have two concerns:
 - #message is non-standard (AFAIK?)
 - this message is not really actionable (unless the user can choose to run on a different CPU architecture), will likely end up being felt as spam.

Happy to merge the came CL just without the #message.